test/verify/check-selinux

```diff
@@ -141,6 +141,11 @@ class TestSelinux(testlib.MachineCase):
         b.wait_not_present(row_selector)
         # unbreak sebool, and trigger the error again
         m.execute(f"chmod a+x {sebool_path}; rm -r ~admin/public_html")
+        if m.image.startswith("rhel-8"):
+            # HACK: RHEL 8's setroubleshootd sometimes misses the second alert
+            m.execute("systemctl restart setroubleshootd")
+            b.reload()
+            b.enter_page("/selinux")
```

Comment:
Do we have an issue for this?

Reply:
No. It's RHEL 8 only (which is frozen) and terribly hard to reproduce. I.e. no reproducer that would make an useful bug report, and not "exception" material either.